src/plugins/discover/public/plugin.tsx

```diff
@@ -354,6 +373,7 @@ export class DiscoverPlugin
       history: this.historyService.getHistory(),
       urlTracker: this.urlTracker!,
       profilesManager,
+      ebtContextManager: profilesManager.ebtContextManager,
```

Comment:
Rather than making `profilesManager.ebtContextManager` public, could we instead make it private and just pass `ebtContextManager` to `getDiscoverServices`? It feels like the EBT manager should be just an implementation detail of the profiles manager.